app/globals.css

```
@@ -226,33 +218,32 @@
   background-color: #0a0a0a;
   background-color: var(--background);
   border-color: var(--border);
-  box-shadow: 0 12px 30px rgba(15, 23, 42, 0.08);
+  box-shadow: 0 12px 30px rgb(15 23 42 / 8%);
 }
 
 .navbar[data-nav-state="mobile-open"] {
   background-color: #0a0a0a;
   background-color: var(--background);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)